docs/configuration/https.md

```diff
@@ -0,0 +1,65 @@
+# HTTPS and authentication
+
+Prometheus supports basic authentication and TLS.
+This is **experimental** and might change in the future.
```

Comment:
Let's mention it on the stability page too, i'm expecting this is be permanently experimental given security stuffs.

Reply:
Oh, I was thinking that the stability page was in the docs repo. Now updated.

Reply:
It's for Prometheus only, so in this repo.